features/admin.roles.v2/pages/role.tsx

```diff
@@ -221,27 +221,25 @@ const RolesPage: FunctionComponent<RolesPagePropsInterface> = (
     return (
         <PageLayout
             action={
-                !isSubOrg && !isRolesListLoading && (rolesList?.totalResults > 0)
-                    ? (
-                        <Show when={ featureConfig?.userRoles?.scopes?.create }>
-                            <PrimaryButton
-                                data-componentid={ `${componentId}-add-button` }
-                                onClick={ () => handleCreateRole() }
-                            >
-                                <Icon
-                                    data-componentid={ `${componentId}-add-button-icon` }
-                                    name="add"
-                                />
-                                { t("roles:list.buttons.addButton", { type: "Role" }) }
-                            </PrimaryButton>
-                        </Show>
-                    ) : null
+                (
```

Comment:
we can still keep `!isRolesListLoading && (rolesList?.totalResults > 0`, can't we?

Reply:
If we keep that logic, the `Add Role button` will only be visible when there are roles to be visible in the sub organization level. But there can be cases where there are no roles in the sub organization level. In that case also we need to display the `Add Role button`.